app/src/main/java/com/hms/referenceapp/photoapp/ui/listuser/ListUserViewModel.kt

```diff
@@ -98,44 +97,49 @@ class ListUserViewModel @Inject constructor(
                 savedUserList = currentUserList
             )
         }
-        Log.d("TAG", "message : ${getSelectedUsers()}")
     }
 
     fun controlFileInformationModel(fileInformation: FileInformationModel) {
         if (fileInformation.description.isEmpty()) {
             _listUserUiState.update { currentListUserUiState ->
                 currentListUserUiState.copy(
-                    error = "description can not be empty"
+                    error = ERROR_DESCRIPTION
                 )
             }
             return
         }
         if (fileInformation.title.isEmpty()) {
             _listUserUiState.update { currentListUserUiState ->
                 currentListUserUiState.copy(
-                    error ="title can not be empty"
+                    error = ERROR_TITLE
                 )
             }
             return
         }
         if (fileInformation.userList.isEmpty()) {
             _listUserUiState.update { currentListUserUiState ->
                 currentListUserUiState.copy(
-                    error = "You have to select at least one user"
+                    error = ERROR_USER_LIST
                 )
             }
             return
         }
-          _listUserUiState.update { currentListUserUiState ->
-              currentListUserUiState.copy(
-                  shareImageInformationFileTaken = true
-              )
-          }
+        _listUserUiState.update { currentListUserUiState ->
+            currentListUserUiState.copy(
+                shareImageInformationFileTaken = true
+            )
+        }
     }
 
     fun errorShown() {
         _listUserUiState.update {
             it.copy(error = null)
         }
     }
+
+    companion object{
```

Comment:
error messages should extract to resources for future usage, translation etc. So you can get from resources using provider inside viewmodel